newdle/client/src/components/RecipientList.js

```diff
@@ -56,7 +57,7 @@ export default function RecipientList({recipients, color, icon}) {
           }}
           circular
         >
-          {`+${recipients.length - DEFAULT_RECIPIENTS_SHOWN} more`}
+          <Trans>{`+${recipients.length - DEFAULT_RECIPIENTS_SHOWN} more`}</Trans>
```

Comment:
This needs pluralization since in other languages it may depend on how many more there are to show.

```suggestion
          <Plural
            value={recipients.length - DEFAULT_RECIPIENTS_SHOWN}
            one={'+# more'}
            other={'+# more'}
          />
```